ygot/schema_tests/schema_benchmark_test.go

```diff
@@ -0,0 +1,14 @@
+package schematest
+
+import (
+	"testing"
+
+	"github.com/openconfig/ygot/exampleoc"
+)
+
+func BenchmarkPopulateDefaults(b *testing.B) {
```

Comment:
Thanks for adding the benchmark -- how does this perform?

Reply:
BenchmarkPopulateDefaults-12 18048 67670 ns/op

Seems to run within 100 us.